.babelrc

```diff
@@ -0,0 +1,9 @@
+{
+  "presets": [
+    "es2015-node4"
+  ],
+  "plugins": [
+    "transform-runtime",
```

Comment:
OK, this was generated by https://github.com/Springworks/generator-springworks/tree/master/generators/babel-builder. Should transform-runtime be included at all?